### ISSUE TITLE
Creating empty layers

### ISSUE THREAD

**Issue by mechtaev** [modus-continens/modus#201]
We can consider providing a predicate that creates an empty layer, say `empty_layer` or `skip`:

```
install_dependencies("debian:bullseye") :- run("apt-get install iproute2").
install_dependencies("debian:bullseye-slim") :- skip.

```

A concrete use case is here: https://github.com/modus-continens/openjdk-images-case-study/blob/fa02585eac8676b69918ad0dc53976be33ab29b4/linux.Modusfile#L59

**Comment by maowtm** [comment]
Actually for that we just need something that acts like a layer expression but creates no layer, rather than introducing redundant layers in the image

**Comment by mechtaev** [comment]
Yes, by saying "empty layer" I meant no layers, sorry for not being precise.

**Comment by thevirtuoso1973** [comment]
e.g.
```dockerfile
FROM alpine
RUN []
```
does not produce a run layer. So maybe `run("")` should also not create a layer.

**Comment by mechtaev** [comment]
I tried runnig 

```Dockerfile
FROM alpine
RUN []
RUN sleep 1s
```
and then

```Dockerfile
FROM alpine
RUN sleep 1s
```

The sleep command was not cached, so seems like it still creates a layer.
